app/assets/javascripts/app/application/uploader.js.coffee

```diff
@@ -101,6 +101,11 @@ convertSize = (str) ->
   else
     null
 
-getFormatsFromFiles = (files) ->
-  if files? && files == 'image/*'
-    ['jpg', 'jpeg', 'png']
\ No newline at end of file
+getFormatsFromAccept = (accept) ->
+  if accept? && accept == 'image/*'
+    ['jpg', 'jpeg', 'png']
+
+# Use this when firefox is ready to use the accepts='.jpg,.png, ...' (version 37 maybe)
```

Comment:
Maybe, it's just that this should be standard html5 and it works in chrome and mostly works on firefox (right now it will list some types of images like .svg and .tiff but won't let you select it (a check done by the js code)). So I'm in favor of just waiting a little until this lands on firefox.